config/icons.json

```diff
@@ -21,6 +21,7 @@
   "FaTwitter": "#00ACEE",
   "FaVimeo": "#1AB7EA",
   "FaYoutube": "#FF0000",
+  "FaRocketchat" : "#FF0000",
```

Comment:
This should be a separate git commit and I am pretty sure you should raise an issue for this first and get approval for the same

Reply:
I have created the issue for the same